berserk/clients/puzzles.py

```diff
@@ -78,3 +78,171 @@ def create_race(self) -> PuzzleRace:
         """
         path = "/api/racer"
         return cast(PuzzleRace, self._r.post(path))
+
+    def next(self, angle: str = "mix", difficulty: Any = "normal"):
```

Comment:
angle is internal name, `theme` would more understandable. Also the parameters should be typed but not checked at runtime, so that users can override if the lib is outdated. they should be put in the types folder.

example https://github.com/lichess-org/berserk/blob/f8fac03542936594d431e96c33ea9e6f324911cc/berserk/types/challenges.py#L8